keylime/tpm/tpm_util.py

```diff
@@ -434,3 +435,47 @@ def crypt_kdfe(
         size -= hashfunc.digest_size
 
     return result[:size_in_bytes]
+
+
+class TPM_Utilities:
```

Comment:
I think we can drop the class wrapper all together.

Reply:
Yeah, it doesn't buy much. Dropping it.